src/terminal/terminal/display.h

```diff
@@ -319,6 +319,12 @@ void guac_terminal_display_resize(guac_terminal_display* display, int width, int
 /**
  * Flushes all pending operations within the given guac_terminal_display.
  */
+void guac_terminal_display_flush_operations(guac_terminal_display* display);
```

Comment:
Please document the `display` parameter.

There is still old documentation in our codebase that omits this (like `guac_terminal_display_flush()` below), but we are including this documentation going forward at least for new/modified code.